components/item/ItemHero.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <div>
+  <div class="item-hero">
     <AwesomeSwiper
       :europeana-identifier="identifier"
       :media="media"
       @select="selectMedia"
     />
-    <b-container class="d-flex justify-content-around mt-5">
+    <b-container class="d-flex justify-content-around media-bar">
```

Comment:
When you look at an item page with all three buttons: rights, download and share, there is space between them. According to the design, the download and share should be on the right and the rights statement on the left. 

An example: item-new/117/HKMS000005_00000ekd

Reply:
yes that's true, have fixed styling